yt_dlp/extractor/acfun.py

```diff
@@ -0,0 +1,194 @@
+import re
+from pathlib import Path
+
+from .common import InfoExtractor
+from ..utils import (
+    float_or_none,
+    int_or_none,
+    traverse_obj,
+)
+
+
+class AcFunVideoIE(InfoExtractor):
+    IE_NAME = 'AcFunVideoIE'
+    _VALID_URL = r'''(?x)
+                    https?://
+                        (?:
+                            www\.acfun\.cn/v/
+                        )
+                        ac(?P<id>[_\d]+)
+                    '''
+
+    _TESTS = [{
+        'url': 'https://www.acfun.cn/v/ac35457073',
+        'info_dict': {
+            'id': '35283657',
+            'title': '【十五周年庆】文章区UP主祝AcFun生日快乐！',
+            'duration': 455.21,
+            'timestamp': 1655289827,
+        },
+        'params': {
+            # m3u8 download
+            'skip_download': True,
+        },
+        'skip': 'Geo-restricted to China',
+    }, {
+        'url': 'https://www.acfun.cn/v/ac35457073',
+        'only_matching': True,
+    }]
+
+    @staticmethod
+    def parse_codec(codec_str):
+        vcodec = None
+        acodec = None
```

Comment:
This line can be simplified like this
`vcodec, acodec = None, None`

Reply:
> This line can be simplified like this `vcodec, acodec = None, None`

ok